devicetypes/smartthings/ct100-thermostat.src/ct100-thermostat.groovy

```diff
@@ -657,7 +659,8 @@ def getSupportedModes() {
 def switchFanMode() {
 	def currentMode = device.currentValue("thermostatFanMode")
 	def supportedFanModes = state.supportedFanModes
-	if (supportedFanModes) {
+	// Old version of supportedFanModes was as string, make sure it gets updated
+	if (supportedFanModes && supportedFanModes[0].size() > 1) {
```

Comment:
Same comment as above about supportedFanModes could be an empty list?  It will it always contain at least "Off"?